vra/resource_fabric_network_vsphere.go

```diff
@@ -109,6 +109,10 @@ func resourceFabricNetworkVsphere() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"custom_properties": {
+				Type:     schema.TypeMap,
+				Computed: true,
```

Comment:
The custom property is optional when creating the zone. Please refer swagger doc. So, it will be both computed and optional true